dashboard/app/graphs.go

```diff
@@ -419,15 +427,19 @@ func extractMetric(stat *ManagerStats, metric string) float64 {
 	}
 }
 
-func createCheckBox(r *http.Request, caption string, values []string) *uiCheckbox {
+func createCheckBox(r *http.Request, caption string, values []string) (*uiCheckbox, error) {
```

Comment:
Need a comment here explaining what the function does and in which case it returns errors.

Reply:
I added error generation explanation.
But I'm not sure we want to explain all the private functions by rich comments. It could be a quite sensitive overhead for us.
Do you think we need rich comments for private functions as a general requirement?

Reply:
> I added error generation explanation.
Looks good, thanks!

> Do you think we need rich comments for private functions as a general requirement?
I don't think so, but at least it's good to know what a function does without looking at the code.

Reply:
Agree. Naming matters...